api/edgeproto/appinst.proto

```diff
@@ -224,6 +226,8 @@ message AppInst {
   string obj_id = 49 [(protogen.backend) = true, (protogen.hidetag) = "nocmp"];
   // Annotations
   map<string, string> annotations = 52;
+  // database version model ID
```

Comment:
It's in case the upgrade function is run again, or let's say there are two instances of the regional controller and they get restarted one after the other, then the second controller will run the upgrade function that the first controller already ran. This lets us know the object was already upgraded.